ovn-tester/ovn_tester.py

```diff
@@ -380,14 +389,37 @@ def run_test_cluster_density(ovn, cfg):
         for ns in all_ns:
             ns.unprovision()
 
+def run_test_np_cross(ovn, cfg):
+    all_ns = []
+
+    with Context('np_cross_startup', brief_report=True) as ctx:
+        ports = ovn.provision_ports(cfg.pods_ns_ratio*cfg.n_ns)
+        for i in range(cfg.n_ns):
+            ns = Namespace(ovn, f'NS_{i}')
+            ns.add_ports(ports[i*cfg.pods_ns_ratio :
+                               (i + 1) *cfg.pods_ns_ratio - 1])
+            all_ns.append(ns)
```

Comment:
Shouldn't we also add ACLs in the "startup" stage too?

Reply:
@dceara what do you mean? I was thinking the real test would be to add acls? Are we expected to ping at the end?

Reply:
@LorenzoBianconi Ah, right, sorry I misinterpreted what you meant by "startup" here.

Reply:
@dceara are we expected to ping the ports after configuring the ACLs?